src/main/java/dinkplugin/notifiers/MetaNotifier.java

```diff
@@ -64,6 +64,11 @@ public void onGameState(GameState oldState, GameState newState) {
         if (oldState == GameState.LOGGING_IN && newState == GameState.LOGGED_IN) {
             loginTicks.set(INIT_TICKS);
         }
+        // check if the oldState is any that can be considered "in game", and if the new state is "LOGIN_SCREEN"
+        if ((oldState == GameState.LOGGED_IN || oldState == GameState.CONNECTION_LOST || oldState == GameState.LOADING)
```

Comment:
```suggestion
        if ((oldState == GameState.LOGGED_IN || oldState == GameState.CONNECTION_LOST || oldState == GameState.HOPPING)
```

this notifier will never see `LOADING` (filtered out in DinkPlugin), but getting disconnected mid-hop may be a possibility

Reply:
Good catch